lexibank_chenhmongmien.py

```diff
@@ -1,15 +1,14 @@
-import csv
+import collections
+from pathlib import Path
 
-import requests
 from bs4 import BeautifulSoup
-from clldutils.misc import slug
-from clldutils.path import Path
 from clldutils.text import strip_brackets, split_text
 from pylexibank.dataset import NonSplittingDataset
 from tqdm import tqdm
-from pylexibank.dataset import Concept, Language
+from pylexibank import Concept, Language
```

Comment:
But the real path is `pylexibank.models`, right?

Reply:
Yes, but I tend towards exposing what is considered public API on the package level nowadays. This makes it easier on the importing side to just import the whole package and then have more readable code:
```python
import pylexibank

class HLexeme(pylexibank.Lexeme):
    ...


class Dataset(pylexibank.Dataset):
    ....


```

Reply:
Okay, will try to follow that.